tests/tests.mk

```diff
@@ -1,4 +1,13 @@
-export TIMEOUT ?= 30m
+export TIMEOUT ?= 40m
+
+# gocache disabled by -count=1
+# tests in different packages forced to be sequential by -p=1
+%.test: init | $(JUNIT_REPORT)
```

Comment:
I'd also be inclined to prefix with `doc.` like the others. Otherwise we could remove doc. from the front of help since doc.test will go away.